stl/inc/xmemory

```diff
@@ -2066,6 +2066,30 @@ typename _Container::size_type _Erase_nodes_if(_Container& _Cont, _Pr _Pred) {
     return _Old_size - _Cont.size();
 }
 
+template <class _Ty1, class _Ty2>
+void _Deduce_as_pair(const pair<_Ty1, _Ty2>&);
+
+template <class _Ty, class = void>
+_INLINE_VAR constexpr bool _Cannot_as_pair = true;

```

Comment:
Why can't we a verb here? It will variable name a bit longer, but I think we can it.

Reply:
Thinks for pointing it out. I've changed the name to `_Is_not_deducible_as_pair`.